src/test/util/setup_common.h

```diff
@@ -64,6 +65,14 @@ struct BasicTestingSetup {
     util::SignalInterrupt m_interrupt;
     node::NodeContext m_node; // keep as first member to be destructed last
 
+    FastRandomContext m_rng;
+    /** Seed the internal global RNG state and m_rng for testing. This affects all randomness, except GetStrongRandBytes(). */
+    void SeedRandomForTest(SeedRand seed = SeedRand::SEED)
+    {
+        SeedRandomStateForTest(seed);
+        m_rng.Reseed(GetRandHash());
```

Comment:
I understand this was the behavior in the past too, but should we do something else with `m_rng` if the function was called with `SeedRand::ZEROS`?